Modules/Registration/Common/include/itkEuclideanDistancePointMetric.h

```diff
@@ -90,7 +90,9 @@ class ITK_TEMPLATE_EXPORT EuclideanDistancePointMetric
 
   /** Get the derivatives of the match measure. */
   void
-  GetDerivative(const TransformParametersType & parameters, DerivativeType & Derivative) const override;
+  GetDerivative(const TransformParametersType & itkNotUsed(parameters),
+                DerivativeType &                itkNotUsed(derivative)) const override
+  {}
```

Comment:
It appears that `EuclideanDistancePointMetric::GetDerivative` has always been empty, looking at commit 06e94a207e0b9788dc7269720eb8c5ba39155997 (10 Oct 2004):
https://github.com/InsightSoftwareConsortium/ITK/blob/06e94a207e0b9788dc7269720eb8c5ba39155997/Code/Algorithms/itkEuclideanDistancePointMetric.txx#L145-L150

Just curious, was it still meant to be implemented? Or was it intentionally left empty?